python/tvm/relay/op/nn/nn.py

```diff
@@ -3646,3 +3646,38 @@ def batch_to_space_nd(data, block_shape, crops):
     """
 
     return _make.batch_to_space_nd(data, block_shape, crops)
+
+
+def im2col(data, kernel_size, dilation, padding, stride):
+    r"""im2col.
+
+    This operator convert 4-D NCHW data with im2col for PyTorch unfold.
+
+    Please reference
+    `https://pytorch.org/docs/stable/generated/torch.nn.Unfold.html#torch.nn.Unfold`
+
```

Comment:
PyTorch specific stuff shouldn't be in this file. Please explain what this function does instead.

Reply:
If you click "[SOURCE]" link, you will find
```
    def forward(self, input: Tensor) -> Tensor:
        return F.unfold(input, self.kernel_size, self.dilation,
                        self.padding, self.stride)

```
torch.nn.Unfold forward will all F.unfold, F.unfold will call im2col. If you call F.unfold or nn.Unfold forward, from_pytorch will give you a exception.

Reply:
Removed usefulness comments in source code.

Reply:
Please just remove references to PyTorch and describe what it does. This is not frontend code.